libmscore/cmd.cpp

```diff
@@ -2608,6 +2608,7 @@ void Score::cmdImplode()
       Segment* endSegment = selection().endSegment();
       Measure* startMeasure = startSegment->measure();
       Measure* endMeasure = endSegment ? endSegment->measure() : lastMeasure();
+      Q_ASSERT(startMeasure && endMeasure);
```

Comment:
I added this assert just as a sanity check...don't know if it is even needed...but who knows...if someone introduces a bug somewhere else at least will crash with a somewhat useful message on console.